docs/src/components/base/spacing-scale/README.md

```diff
@@ -0,0 +1,78 @@
+# Spacing scale
+
+The cloud.gov spacing scale is based on a base grid of **8px**. Use even multiples of this base grid when setting the height, width, padding, and margin of blocks and components. When possible, any spacing value should be a multiple of this base grid.
+
+
+## Sass variables
+
+Our spacing grid sass variables provide an easy way to access grid multiples. The variable outputs the final value in terms of `rem` but these are meant to directly correspond to a pixel multiplier.
+
+```
+$grid-05:            0.5 * base     4px
+$grid-1:             1 * base       8px
+$grid-105:           1.5 * base     12px
+$grid-2:             2 * base       16px
+$grid-3:             3 * base       24px
+$grid-4:             4 * base       32px
+$grid-5:             5 * base       40px
+$grid-6:             6 * base       48px
+$grid-7:             7 * base       56px
+$grid-8:             8 * base       64px
+$grid-9:             9 * base       72px
+$grid-10:            10 * base       80px
+```
```

Comment:
This might present better in a markdown table (I haven't seen how our fractal theme presents them).
```
| Class | Scale | Computed |
| -------- | ------- | -------------- |
| `$grid-05` | 0.5 * base | 4px |
```

| Class | Scale | Computed |
| -------- | ------- | -------------- |
| `$grid-05` | 0.5 * base | 4px |